core/domain/state_domain.py

```diff
@@ -1283,7 +1295,14 @@ def validate(self):
 
     @classmethod
     def create_default_subtitled_html(cls, content_id):
-        """Create a default SubtitledHtml domain object."""
+        """Returns a default SubtitledHtml domain object.
+
+        Args:
+            content_id: str. A unique id referring to the content field.
```

Comment:
It's not clear what "unique" means here, since there's only one ID being passed in (and IDs are typically unique by definition). Suggest just saying "An ID" or, better, "The ID".

Reply:
Corrected, Done.